assets/media-gallery.js

```diff
@@ -46,12 +46,16 @@ if (!customElements.get('media-gallery')) {
 
         this.preventStickyHeader();
         window.setTimeout(() => {
-          if (this.elements.thumbnails) {
-            activeMedia.parentElement.scrollTo({ left: activeMedia.offsetLeft });
-          }
-          if (!this.elements.thumbnails || this.dataset.desktopLayout === 'stacked') {
-            activeMedia.scrollIntoView({ behavior: 'smooth' });
+          if (this.dataset.enableStickyInfo === 'false'
+            || !this.mql.matches
+            || (this.dataset.enableStickyInfo === 'true' && this.dataset.desktopLayout === 'stacked')
+          ) {
+            const gallery = document.querySelector('[id^="GalleryViewer"]');
+            const galleryRect = gallery.getBoundingClientRect();
+            const top = galleryRect.top + window.scrollY;
+            window.scrollTo({ top: top, behavior: 'smooth' });
           }
+          if (this.elements.thumbnails) activeMedia.parentElement.scrollTo({ left: activeMedia.offsetLeft });
```

Comment:
I think the issue is still there 🤔 If I try a product with 3 variant images and nothing else, I get the same scrolling up the info issue: [video](https://screenshot.click/05-09-5k7il-eb1ea.mp4)

Reply:
Okay, it seems I over simplified for the commit and picked the wrong option out of 2 that I was playing with.
I've now changed it to scroll to the section instead of the Gallery viewer.